db/migrate/20181017152837_shotgun.rb

```diff
@@ -82,51 +82,39 @@ def change
       t.index :active_storage_attachment_id
     end
 
-    create_range_partition :impressions, id: :uuid, default: -> { "gen_random_uuid()" }, partition_key: "displayed_at_date" do |t|
```

Comment:
Maybe ad note to README that you are re-consolidating the shotgun. Then we can remove the note once we stop doing that

Reply:
Decided not to do this